bin/update-wasm-grammars.sh

```diff
@@ -4,9 +4,12 @@ set -eu
 
 cd web-playground/public/scripts/
 
-WORKDIR="$(mktemp -d)"
+# We don't use mktemp because <SOMETHING I CAN'T REMEMBER> doesn't work
+# across disparate filesystems
+WORKDIR=./tmp/
+mkdir -p $WORKDIR
```

Comment:
@ErinvanderVeen I don't remember the details :sweat_smile: Something to do with `rename`, rather than `mv`; I assume that's in the Rust code somewhere...

Reply:
tree-sitter

<3